apps/condo/domains/common/components/Modal.tsx

```diff
@@ -1,6 +1,7 @@
 import styled from '@emotion/styled'
 import { Modal as DefaultModal } from 'antd'
 
+/** @deprecated use Modal from @open-condo/ui */
 export const Modal = styled(DefaultModal)`
```

Comment:
Aren't you removed all usage in this PR?)

Reply:
removed it